core/src/main/java/org/apache/brooklyn/entity/group/DynamicCluster.java

```diff
@@ -148,6 +148,11 @@
     ConfigKey<EntityFactory> FACTORY = ConfigKeys.newConfigKey(
             EntityFactory.class, "dynamiccluster.factory", "factory for creating new cluster members", null);
 
+
+    @SetFromFlag("removalStrategy")
+    ConfigKey<RemovalStrategy> CONFIGURABLE_REMOVAL_STRATEGY = ConfigKeys.newConfigKey(RemovalStrategy.class, "dynamiccluster.configurable.removalStrategy");
+
+    @Deprecated /** @deprecated since 0.9.0; use {@link CONFIGURABLE_REMOVAL_STRATEGY instead} */
```

Comment:
`{@link CONFIGURABLE_REMOVAL_STRATEGY instead}` should be `{@link CONFIGURABLE_REMOVAL_STRATEGY} instead`.